.github/workflows/helm-release.yaml

```diff
@@ -20,7 +20,12 @@ jobs:
           # Enable pipefail so git command failures do not result in null versions downstream
           set -x
 
-          echo ::set-output name=VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
+          echo ::set-output name=RELEASE_VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
```

Comment:
this sets a job output, but it's used as an environment variable later, so I'm not sure this actually works

you might have to write this to the $GITHUB_ENV file if you expect to use it as an env var

Reply:
it works! - https://github.com/PrefectHQ/prefect-helm/actions/runs/3102476140/jobs/5024809170